bin/ParseDarwinSamplesheet.sh

```diff
@@ -48,8 +48,8 @@ function showHelp() {
 ======================================================================================================================
 Scripts to make automatically a samplesheet for the concordance check between ngs and array data and pushes the ngs and 
 array data to the destination machine.
-ngs.vcf should be in /groups/${NGSGROUP}/${PRM_LFS}/concordance/ngs/.
-array.vcf should be in /groups/${ARRAYGROUP}/${PRM_LFS}/concordance/array/.
+ngs projects should be in /groups/${NGSGROUP}/${PRM_LFS}/projects.
+array projects should be in /groups/${ARRAYGROUP}/${PRM_LFS}/openarray/.
```

Comment:
Dat geldt alleen voor de OpenArray projects. Is het nog zo te maken dat dit ook werkt voor GSA of andere arrays?

Reply:
Dat kan als je de array naar een vcf converteert en een juiste bedfile en jobfile aanlevert kan de concordance pipeline dat prima. Maar dat gaan niet vanzelf en in nu ook niet iets wat voor validatie binnen de scope valt. En daar hebben we ook al een gevalideerde pipeline voor draaien voor de tijd dat die GSA nog gebruikt wordt.